doc/version_history.rst

```diff
@@ -2,9 +2,13 @@
 Version History
 ===============
 
-v0.4.0
+v0.4.1
```

Comment:
I think we can keep the version as 0.4.0 until we release it officially.

Reply:
Done in the commit: 46ac89d. Thanks!